### ISSUE TITLE
Possible typo on `animate` API documentation

### ISSUE THREAD

**Issue by jordanmoore753** [jquery/api.jquery.com#1157]
Hello! There may be a possible typo on the `animate` API documentation.

In the example shown, the code is:

```html
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>animate demo</title>
  <style>
  div {
    background-color: #bca;
    width: 100px;
    border: 1px solid green;
  }
  </style>
  <script src="https://code.jquery.com/jquery-3.4.1.js"></script>
</head>
<body>
 
<button id="go">&raquo; Run</button>
<div id="block">Hello!</div>
 
<script>
// Using multiple unit types within one animation.
 
$( "#go" ).click(function() {
  $( "#block" ).animate({
    width: "70%",
    opacity: 0.4,
    marginLeft: "0.6in",
    fontSize: "3em",
    borderWidth: "10px"
  }, 1500 );
});
</script>
 
</body>
</html>
```
However, the next paragraph explaining it says that:

> The first button shows how an unqueued animation works. It expands the div out to **90% width** while the font-size is increasing. 

It is my understanding that the `animate` method expands the `div` out to 70% width, since the value for the `width` property in the **properties** object is **70%**.

Thank you for reading this and have a great day.

**Comment by aldimar-junior** [comment]
Taking a look at the documentation the paragraph saying:

> The first button shows how an unqueued animation works. It expands the div out to 90% width while the font-size is increasing. Once the font-size change is complete, the border animation will begin. The second button starts a traditional chained animation, where each animation will start once the previous animation on the element has completed.

Is referring to the examples right below it, not the one above, if you take a look at the example below the paragraph it actually expands the div out to 90% width:

```html
<script>
$( "#go1" ).on( "click", function() {
  $( "#block1" )
    .animate({
      width: "90%"
    }, {
      queue: false,
      duration: 3000
    })
    .animate({ fontSize: "24px" }, 1500 )
    .animate({ borderRightWidth: "15px" }, 1500 );
});
</script>
```

It might be a good thing to change the order of the paragraphs/examples for better clarity and/or just move the mentioned paragraph and position it below the "correct" example.

**Comment by aldimar-junior** [comment]
Probably appending "In the next example" i.e:

> **In the next example**, the first button shows how an unqueued animation works. It expands the div out to 90% width while the font-size is increasing. Once the font-size change is complete, the border animation will begin. The second button starts a traditional chained animation, where each animation will start once the previous animation on the element has completed.

would increase clarity and remove the ambiguity of "is this text referring to the example above? or the one below?"

**Comment by mgol** [comment]
I don't think that's the correct approach; we'd have to add something like that to every single example.

I'd rather tweak the `<example>` definition to expand to something that will be clearer about the intent.

The `<example>` tag rendering is defined in `grunt-jquery-content`; it's an XSLT transform. Examples handling is here: https://github.com/jquery/grunt-jquery-content/blob/1abaf4b81c800f332880583e6d97b0ef6b2b44c0/tasks/jquery-xml/entries2html-base.xsl#L95-L113

Maybe we should have a header (h3?) for each example with the counter (Example 1, Example 2, etc.) which would separate them more clearly?

**Comment by aldimar-junior** [comment]
Yeah, sure, that would be a great change, I'll take a look at the files you mentioned and see if I can make the changes

**Comment by aldimar-junior** [comment]
Done @mgol PR Nº 95, I'll close my previous PR and I'm waiting for your approval.

**Comment by mgol** [comment]
This has been fixed and examples are more clearly separated now. Thanks to @aldimar-junior for the fix!
